code/modules/SCP/SCPs/SCP-409.dm

```diff
@@ -0,0 +1,39 @@
+/obj/structure/scp409/
+	name = "SCP-409"
+	desc = "A white crystal, something beckons you to come and touch it..."
+	icon = 'icons/SCP/scp-409.dmi'
+	icon_state = "409"
+	layer = 2.9
+	anchored = TRUE // makes it not able to be moved
+	density = TRUE // makes it have collisions
+	SCP = /datum/scp/scp_409
+
+/datum/scp/scp_409
+	name = "SCP-409"
+	designation = "409"
+	classification = KETER
+
+/obj/effect/decal/cleanable/residue409
+	name = "crystals"
+	desc = "Some crystals on the floor... is that blood?" // something about blood even though it doesnt have blood on the sprite, blame the spriting team
+	icon = 'icons/SCP/scp-409.dmi'
+	icon_state = "residue409"
+	gender = PLURAL
+
+/obj/structure/scp409/attack_hand(mob/user as mob)
+	visible_message("[user] touches [src].")
+	log_and_message_admins("[user] has touched SCP-409 at [x] [y] [z]") // not sure if co-ordinates are required for the JUMP option to be available but its cool :sunglasses:
+	Crystal(user)
+
+/obj/structure/scp409/proc/Crystal(var/mob/living/user) // defines Crystal() which calls crystaltim(), i am dumb
```

Comment:
why so many useless comments

Reply:
Also, proc name shouldn't start with capital letter

Reply:
I'll fix that.